src/pages/AdminEditUser.jsx

```diff
@@ -0,0 +1,321 @@
+                    checked={userHasRole(signingOfficialRole)}
+                    className='checkbox-inline user-checkbox'
+                    onChange={(e) => roleStatusChanged(e, signingOfficialRole)}
+                  />
+                  <label id='lbl_signing_official' className='regular-checkbox rp-choice-questions' htmlFor='chk_signing_official'>Signing Official</label>
+                </div>
+              </div>
+            </div>
+
+            <div className='form-group'>
+              <div className='col-lg-9 col-lg-offset-3 col-md-9 col-md-offset-3 col-sm-9 col-sm-offset-3 col-xs-8 col-xs-offset-4' style={{ paddingLeft: '30px' }}>
+                <div className='checkbox'>
+                  <input
+                    type='checkbox'
+                    id='chk_admin'
+                    checked={userHasRole(adminRole)}
+                    className='checkbox-inline user-checkbox'
+                    onChange={(e) => roleStatusChanged(e, adminRole)}
+                  />
+                  <label id='lbl_admin' className='regular-checkbox rp-choice-questions' htmlFor='chk_admin'>Admin</label>
+                </div>
+              </div>
+            </div>
+
+            <div className='form-group'>
+              {
+                userHasRole(adminRole) && (
+                  <div
+                    className='col-lg-9 col-lg-offset-3 col-md-9 col-md-offset-3 col-sm-9 col-sm-offset-3 col-xs-8 col-xs-offset-4'
+                    style={{ paddingLeft: '30px' }}
+                  >
+                    <div className='checkbox'>
+                      <input
+                        id='chk_emailPreference'
+                        type='checkbox'
+                        className='checkbox-inline user-checkbox'
+                        // If email preference is TRUE  -> disable checkbox is not checked
+                        // If email preference is FALSE -> disable checkbox is checked
+                        checked={!state.emailPreference}
+                        onChange={emailPreferenceChanged}
+                      />
+                      <label htmlFor='chk_emailPreference' className='regular-checkbox rp-choice-questions bold'>Disable Admin email notifications</label>
+                    </div>
+                  </div>
+                )
+              }
+              <div className='col-lg-12 col-xs-12 inline-block'>
+                <div style={{ marginLeft: '40px' }}>
+                  <button
+                    id='btn_save'
+                    onClick={() => props.history.push('/admin_manage_users')}
+                    className='f-left btn-primary btn-back'
+                  >
+                   Back
+                  </button>
+                </div>
+                <button
+                  id='btn_save'
+                  onClick={OKHandler}
+                  className='f-right btn-primary common-background'
+                  disabled={!displayNameValid}
```

Comment:
I'm noticing that now, the Save button is disabled after an Institution or Role change until the Name value is changed. I'm not sure if this is intentional, but this is different between what's currently in dev.

Reply:
Nice catch!

Reply:
Thank for this catch @aarohinadkarni  It's a bug. I'll send out a fix.